tpm2tools/keys_test.go

```diff
@@ -70,3 +72,184 @@ func BenchmarkStorageRootKeyRSA(b *testing.B) {
 		key.Close()
 	}
 }
+
```

Comment:
I think having the sealing tests in a seperate seal_test.go file would make more sense

Reply:
Done.